meta.go

```diff
@@ -122,6 +122,40 @@ type Storage struct {
 	Properties []string `bson:"properties,omitempty"`
 }
 
+// DeviceType defines a device type.
+type DeviceType string
+
+// DeviceGPUGeneric represents generic GPU device type.
+const DeviceGPUGeneric DeviceType = "gpu"
```

Comment:
Wrap these in a single const block
const (
  // comment
  blah

  // comment
  blah
)

Reply:
We shouldn't encode this sort of validation here in the charm. The charm should just check that device type is not empty. Juju should then later validate the values based on the capabilities of the container substrate. eg we don't want to have to update this library when k8s adds a new device type